### ISSUE TITLE
questions regarding training loss and parser

### ISSUE THREAD

**Issue by chengengliu** [Andrew-Qibin/VisionPermutator#3]
Hi, thanks for your work and sharing the logs. I'm training the model with some data but I can see my training loss is way lower than your loss on the log(like first five epochs drops to 1.0). 
I'm thinking it might be the parser's faults or somewhere else. How do you configure the parser? I'm new to timm and I see your code doesn't configure parser but leaves timm to auto configure it. Any help would be appreciated. Thanks.

**Comment by zihangJiang** [comment]
I think you can refer to https://github.com/Andrew-Qibin/VisionPermutator/blob/0849d7fa9858fabe580ce8b6310a87c037325c09/main.py#L39 for configuring the parser.
The command line for training are provided [here](https://github.com/Andrew-Qibin/VisionPermutator#training).

**Comment by chengengliu** [comment]
> I think you can refer to
> 
> https://github.com/Andrew-Qibin/VisionPermutator/blob/0849d7fa9858fabe580ce8b6310a87c037325c09/main.py#L39
> 
> for configuring the parser.
> The command line for training are provided [here](https://github.com/Andrew-Qibin/VisionPermutator#training).

Thanks zihang, I'll give a try. By the way, what is the minimal scale of dataset that could achieve your model's performance. I notice that your experiments used ImageNet-1k, did you try any datasets that are much smaller that it?

**Comment by zihangJiang** [comment]
All experiments are under the standard ImageNet-1k setting. If you are using a smaller dataset, I think you can finetune from the pre-trained weight.
